activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/rest/ProcessInstanceController.java

```diff
@@ -149,4 +147,10 @@ public ProcessInstanceController(ProcessInstanceRepository processInstanceReposi
                                                                                     pageable),
                                                   processInstanceRepresentationModelAssembler);
     }
+
+    private boolean canRead(ProcessInstanceEntity processInstanceEntity) {
```

Comment:
what I don't like is that the canRead is not giving any clue about the initiator

Reply:
user needs to be the process initiator and have permissions to read the app `securityPoliciesApplicationService.canRead`